Formula/root.rb

```diff
@@ -14,12 +14,17 @@ class Root < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "davix"
```

Comment:
Is davix enabled by default in upstream's builds? @dpiparo

Reply:
Yes, it provides essential features to ROOT for accessing remote files over HTTP (BTW, I'm also part of the core team).